riak_test/src/rtcs.erl

```diff
@@ -411,9 +417,15 @@ riak_bitcaskroot(Prefix, N) ->
 riak_binpath(Prefix, N) ->
     io_lib:format("~s/dev/dev~b/bin/riak", [Prefix, N]).
 
+riak_repl_binpath(Prefix, N) ->
+    io_lib:format("~s/dev/dev~b/bin/riak-repl", [Prefix, N]).
+
 riakcmd(Path, N, Cmd) ->
     lists:flatten(io_lib:format("~s ~s", [riak_binpath(Path, N), Cmd])).
 
+riak_repl_cmd(Path, N, Cmd) ->
+    lists:flatten(io_lib:format("~s ~s", [riak_repl_binpath(Path, N), Cmd])).
+
```

Comment:
Why don't we just use `rtdev:riakreplcmd/3` and `rtdev:run_riak_repl/3`? is that because different harness?